File: codingTest/baekjoon/python/14425.py
# N, M = map(int, input().split())
# S = set()
# for _ in range(N):
#     S.add(input())
# count = 0
# for _ in range(M):
#     if input() in S:
#         count += 1
# print(count)

import sys
N, M = map(int, input().split())
S = {sys.stdin.readline() for _ in range(N)}
count = 0
for _ in range(M):
    if sys.stdin.readline() in S:
        count += 1
print(count)

File: codingTest/baekjoon/javascript/14425.js
const rl = require("readline").createInterface({ input: process.stdin, output: process.stdout });

const input = [];
rl.on("line", line => {
  input.push(line);
}).on("close", () => {
  let [N] = input.shift().split(" ");
  const S = new Set();
  let count = 0;
  while (N--) S.add(input.shift());
  for (const i of input) if (S.has(i)) count++;
  console.log(count);
});
